app/models/concerns/file_backed_model.rb

```diff
@@ -51,8 +51,10 @@ def from_file(filename)
     # Returns a Pathname to location configured in the database through the
     # :setting_name setting
     def pwd
+      return @pwd if defined?(@pwd)
```

Comment:
Let's follow the same format in methodology model: https://github.com/dradis/dradis-ce/blob/master/app/models/methodology.rb#L44

Reply:
Updated.